conda_smithy/cli.py

```diff
@@ -114,9 +92,12 @@ def __call__(self, args):
         msg = 'Initial feedstock commit with conda-smithy {}.'.format(__version__)
 
         try:
+            os.makedirs(feedstock_directory)
```

Comment:
In addition to ensure there is a directory to create a repo in and add stuff to, should add this ensures we don't accidentally stomp on a preexisting feedstock as it will raise an `OSError` if we do.